.eslintrc.json

```diff
@@ -31,6 +31,8 @@
         }
     },
     "rules": {
+        "filenames/match-exported": "off",
+        "class-methods-use-this": "off",
```

Comment:
i think you're using this, no? maybe it was during development?

Reply:
Yeah, my bad